src/react/ui-elements/Veeam/VeeamSummary.tsx

```diff
@@ -188,7 +188,7 @@ export const VeeamSummary = ({
           content={
             <WrapperWithWidth>
               <HideCredential credentials={secretKey} />
-              <Clipboard text={accessKey} />
+              <Clipboard text={secretKey} />
```

Comment:
Shall we add a test for the copy buttons ?

Reply:
Upgrade to UserEvent v14 by the way as well !!!!